pkg/api/client_runtime_test.go

```diff
@@ -136,7 +136,7 @@ func TestCloudClientRuntime_getRuntime(t *testing.T) {
 			args: args{op: &runtime.ClientOperation{
 				PathPattern: "/deployments",
 			}},
-			want: &runtimeclient.Runtime{BasePath: "/api/v1/"},
+			want: &runtimeclient.Runtime{BasePath: "/api/v1"},
```

Comment:
I assumed this was benign, but not sure why it changed

Reply:
Basepath was updated in openapi spec file, because some language generators not generating correct URL's if there was a trailing slash.